spec/requests/session_spec.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Sessions', type: :request do
+  subject(:login_url) { post '/admin', params: { login: params } }
+
+  let!(:user) { create(:user, password: Faker::Lorem.word) }
+  let(:params) do
+    {
+      email: user.email,
+      password: user.password
+    }
+  end
+
+  describe 'login a user' do
+    before do
```

Comment:
You are testing the login here so instead of calling it inside the before block you should call it inside the expectations.

Reply:
It was my idea to move that subject in a before block. 
But i think you are right.